bots/jabber_mirror_backend.py

```diff
@@ -90,15 +96,18 @@ def __init__(self, jid, password, rooms):
         self.register_plugin('xep_0199') # XMPP Ping
 
     def set_zulip_client(self, client):
+        # type: (zulip.Client) -> None
         self.zulip = client
 
     def session_start(self, event):
+        # type: (Any) -> None
```

Comment:
This is pretty reasonable to leave as `Any` for now. The closest I've come to an answer is [this comment](https://github.com/fritzy/SleekXMPP/blob/6e27f28c854ce4ae1d9f0cc8ee407bda8de97d3b/sleekxmpp/xmlstream/xmlstream.py#L1163-L1165) which implies that we're expecting either an empty dictionary or a "stanza object", which also sounds pretty complicated.